dataRetriever/requestHandlers/requestHandler.go

```diff
@@ -150,7 +152,10 @@ func (rrh *resolverRequestHandler) requestHashesWithDataSplit(
 				"batch size", len(batch),
 			)
 		}
+		return err
```

Comment:
Move this return after line 153, or better remove it at all

Reply:
done.